proposals/models.py

```diff
@@ -400,9 +400,10 @@ class Proposal(models.Model):
     )
 
     avg_understood = models.BooleanField(
-        _('Ik heb kennis genomen van het bovenstaande en begrijp mijn verantwoordelijkheden ten opzichte van de AVG.'),
-        default=False,
-        null=False,
+        _('Ik heb mijn aanvraag en de documenten voor deelnemers besproken met de privacy officer.'),
+        default=None,
+        null=True,
+        blank=True,
         validators=[AVGUnderstoodValidator],
     )
```

Comment:
I'm not sure if changing the label/help text of an existing, different question, is the way to go here. You're effectively rewriting history a bit. 

It's less of a problem now that we have canonical PDF, as the record will not change (and thus, the proper history is preserved). 

However, there are still two issues;
From a developer view, the variable name is confusing now. 
From an applicants view, upon revision/copy/amendment we're going to fill in 'yes' to all existing proposals for this question. (As it was required to be true before). This is less than ideal, as ideally we want to force applicants to re-evaluate if they actually did/need to contact the privacy officer. 

I'd remove the whole field and add another; that should also fix your migration error ;)